### DISCUSSION TITLE
Can't rsync to update a local miroir

### DISCUSSION THREAD

**Discussion by simonjulien** [oerdnj/deb.sury.org#1741] [Q&A] [answered]
Hello,

I use the rsync mirroring from packages.sury.org/repositories/php once a week to maintain some local lab machine running PHP.

First thank you for the job because eveything is perfectly working ! Your work is amazing ! Thank you again.

Since 2 weeks maybe more, I can't manage to update my local mirror with an error not really handy to diagnose : 
```
rsync -av --delete rsync://packages.sury.org/repositories/php .
rsync: failed to connect to packages.sury.org (185.93.2.243): Connection timed out (110)
```

Tried .xyz too with the same result. 

Some changes are underway or is there a new way to do this ?

Regards
Julien

**Comment by bashgeek** [comment, marked as answer]
The canonical name is rsync.sury.org: https://www.patreon.com/posts/use-rsync-luke-32563640

_Originally posted by @oerdnj in https://github.com/oerdnj/deb.sury.org/issues/1738#issuecomment-1070365041_

**Reply by simonjulien** [reply]
Well I did not saw this change ! Tested and everything is now alright !

Thank you for your time @bashgeek.

Regards
